builders/maven.go

```diff
@@ -35,6 +39,17 @@ func (m MavenArtifact) Revision() string {
 	return m.Version
 }
 
+// PomFile represents the schema of a common pom.xml file
+type PomFile struct {
```

Comment:
Is `builders.PomFile` meant to be public?

Reply:
Yes, we may want to reuse this among other builders, and potentially move it into `common.go`